index.html

```diff
@@ -0,0 +1,166 @@
+                        <img src="IMAGES/competition.png" alt="presenting" class="p-image">
+                        <h5 class="p-title">Presenting</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            Each student gets to presenet their project every month depending on that milestone's topic.
+                        </p>
+                    </div>
+                </div>
+                <!-- PROGRAM 4 -->
+                <div class="program real-life-experience">
+                    <div class="program-head">
+                        <img src="IMAGES/practice_instruments.png" alt="real-life-experience" class="p-image">
+                        <h5 class="p-title">Real Life Experince</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            In the last three modules pf the program, students start to participate in real life activities depending on their fields.
+                        </p>
+                    </div>
+                </div>
+                <!-- PROGRAM 5 -->
+                <div class="program contests">
+                    <div class="program-head">
+                        <img src="IMAGES/competition.png" alt="contests" class="p-image">
+                        <h5 class="p-title">Contests</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            At the end of every block(3 months), We hold a music contest where students present their creations to the public.
+                        </p>
+                    </div>
+                </div>
+                <!-- PROGRAM 6 -->
+                <div class="program grad-prom">
+                    <div class="program-head">
+                        <img src="IMAGES/practice_instruments.png" alt="Promotion" class="p-image">
+                        <h5 class="p-title">Promotion</h5>
+                    </div>
+                    <div class="p-description">
+                        <p class="p-more">
+                            After students finish their one year program, they graduate. They are also promoted in the public through Darko Music.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="see-more">SEE MORE PROGRAMS</div>
+            <button type="button" class="apply"><strong class="join">Apply Now</strong></button>
+        </section>
+        <!-- SPEAKERS -->
+        <section class="featured-speakers" id="speakers">
+            <div class="title"><h2 class="fp">Featured Speakers</h2></div>
+            <div class="underline"></div>
+            <div class="speakers">
+                <div class="left-side">
+                </div>
+                <div class="right-side"> 
```

Comment:
- [ ] Well done on adding the speakers dynamically, you did a great job but kindly move the items to the center because right now it has more space the right. In this case, you can remove the margins that you gave and you can use margin: auto; and please the background that pictures of the speaker have. Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163674296-19248e37-d21a-4cf0-92c6-64afc2879ba5.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163674302-f2db50d8-44c5-4526-b30f-d31408b6d394.png)